schemes/check/CMakeLists.txt

```diff
@@ -51,15 +51,18 @@ set(SOURCES
     check_noop.f90
 )
 
-#------------------------------------------------------------------------------
-# Add the auto-generated caps
-add_custom_command(
-    OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/check_test_cap.f90
-    DEPENDS ${CCPP_MKCAP}
-    COMMAND ${CCPP_MKCAP} -o ${CMAKE_CURRENT_BINARY_DIR}/check_test_cap.f90
-                             ${CMAKE_CURRENT_SOURCE_DIR}/scheme.xml
-)
-list(APPEND SOURCES ${CMAKE_CURRENT_BINARY_DIR}/check_test_cap.f90)
+# DH the mechanism to create caps from XML is no longer supported; todo:
+# replace with a test that parses a simple metadata table in check_test.f90
+# and creates check_test_cap.f90 - also needs a variable definition file.
+##------------------------------------------------------------------------------
+## Add the auto-generated caps
+#add_custom_command(
+#    OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/check_test_cap.f90
+#    DEPENDS ${CCPP_MKCAP}
+#    COMMAND ${CCPP_MKCAP} -o ${CMAKE_CURRENT_BINARY_DIR}/check_test_cap.f90
+#                             ${CMAKE_CURRENT_SOURCE_DIR}/scheme.xml
+#)
+#list(APPEND SOURCES ${CMAKE_CURRENT_BINARY_DIR}/check_test_cap.f90)
```

Comment:
It used top be useful - check_test_cap.f90 was created on the fly by ${CCPP_MKCAP} = mkcap.py from scheme.xml; however, this code is no longer supported. I want to keep the code there so that we can add a test using the current method (running ccpp_prebuild.py).